cmd/configure.ac

```diff
@@ -1,13 +1,13 @@
-AC_PREREQ([2.69])
-AC_INIT([snap-confine], m4_esyscmd_s([cat VERSION]), [snapcraft@lists.ubuntu.com])
+AC_PREREQ([2.71])
```

Comment:
is AS_HELP_STRING available in older releases? can we change this back to 2.69?

Reply:
I've downgraded and ran a wide test to see if it works.